packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -326,6 +326,8 @@ export function TransferPanelMain({
     })
   }, [isConnectedToArbitrum, externalFrom, externalTo, history])
 
+  const ethBridger = useMemo(() => new EthBridger(l2.network), [l2.network])
```

Comment:
There's no need to do any of these changes, as we are calling methods from `useArbTokenBridge`. The only change we need to make is not pass in the signer.

Reply:
true, just noticed